### DISCUSSION TITLE
Icon request: icon-gpu

### DISCUSSION THREAD

**Discussion by othomasj** [FortAwesome/Font-Awesome#21112] [Icon Requests]
Would be very useful to have for system dashboards.

![image](https://user-images.githubusercontent.com/23376185/39456356-ca884ca8-4c9a-11e8-8c94-9129323979c7.png)

**Comment by yovantichrist** [comment]
+1

**Comment by TexasSwede** [comment]
I agree, I would also like to see a motherboard icon, a processor icon (yes, I know there is a microchip icon https://fontawesome.com/icons/microchip?style=light , and a mouse icon.

**Comment by teomanofficial** [comment]
it have been a year.. :/

**Comment by raiguard** [comment]
+1 from me. I would like to have a fontawesome version of the GPU Icon I am using in my project, since almost absolutely every other icon I am using is fontawesome.

![image](https://user-images.githubusercontent.com/3515394/58041972-acbb2480-7af6-11e9-96b8-701567697803.png)

**Comment by TexasSwede** [comment]
This icon would be useful to me as well.

**Comment by m0s000** [comment]
+1

**Comment by samuelbryant** [comment]
+1 I know it's old but I would love this

**Comment by mario-gabriel-campos** [comment]
definitely +1

**Comment by chiliwax** [comment]
+1

**Comment by rodrigo-ceccato** [comment]
+1

**Comment by Atralb** [comment]
Seems like the team @davegandy @tagliala @robmadole doesn't care.

What fonts are you guys using in the meantime for these kind of things ?

**Comment by enlznep** [comment]
+1

**Comment by diademoff** [comment]
+1

**Comment by pipin77** [comment]
very useful +1

**Comment by CEbbinghaus** [comment]
I can't believe this isn't in yet. I would be keen to see all parts of a PC represented, (GPU, CPU, RAM, Motherboard, Fans, Case, PSU)

+1

**Comment by jmb05** [comment]
+1

**Comment by ghost** [comment]
100% +1

**Comment by Marks12520** [comment]
Yes please

**Comment by Gandharikos** [comment]
+1

**Comment by maxheyer** [comment]
+1

**Comment by teomanofficial** [comment]
We are waiting for years :D

**Comment by CHOWY0152** [comment]
+1

**Comment by jherman** [comment]
How is it that we have `child-combatant` but we can't even get a GPU icon?

**Comment by seanidzenga** [comment]
+1

**Comment by IbolDev** [comment]
+1

**Comment by NikilXYZ** [comment]
+1 and also would like an Nvidia Icon.

**Comment by itsdevdom** [comment]
We definitely need icons for all kinds of PC parts!

**Comment by crshqd** [comment]
+1 could use this for a whole bunch of projects

**Comment by VehementHam** [comment]
+1

**Comment by jeromeluciano** [comment]
+1

**Comment by clsv** [comment]
please add it :-)

**Comment by fourm1** [comment]
Yep

**Comment by mrtx99** [comment]
+1

**Comment by MarkSmersh** [comment]
+1

**Comment by aristotaloss** [comment]
What a joke. The irony that v6 brought `dumpster-fire` but the project still fails to add some of the most basic icons 😅

**Comment by TNTRedstone** [comment]
+1 x infinity, this is my neofetch:
![20240217_18h31m47s_grim](https://github.com/FortAwesome/Font-Awesome/assets/73077470/d9cf9ccd-028f-4f22-a826-90125da54e8b)
it is uneven just because of this

**Comment by ghost** [comment]
+1

**Comment by CelestialCrafter** [comment]
+1

**Comment by bnanchen** [comment]
+1

**Comment by VehementHam** [comment]
Okay, someone should make a fork of this project that has this.

**Comment by bjsho** [comment]
+1 we need this icon

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’re closing this issue because it’s a duplicate of or shares too much similarity with [this request.](https://github.com/FortAwesome/Font-Awesome/issues/9672)

**Comment by matsydev** [comment]
+1

**Comment by DreamyDuck** [comment]
+1

**Comment by synden** [comment]
It's been a few years now. How's it going?

**Comment by wanderduck** [comment]
Is this ever going to happen???

**Comment by sensibleworld** [comment]
Added to 7.2: https://fontawesome.com/icons/gpu?f=classic&s=solid

**Reply by CEbbinghaus** [reply]
Awesome. Thank you so much

**Comment by tinkerpunktheprol** [comment]
An important part of this question was completely ignored: Why is there a child combatant icon? 

It's in very poor taste. Should we also add an american soldier labeled childcombatant_murderer? Or maybe a bomber labeled wedding_ender? Not sure which joker thought this was a good idea, as I couldn't find a PR for it, but it's quite disgusting. 

Not to mention, it was likely not just approved by someone on the FA team, but added by someone from the team. Based on the need to beg for a GPU icon for almost a decade, plus the missing PR. And then it was ignored by everyone for all that time. Who did this, and why? And can we please remove it?

**Reply by gbhand** [reply]
`child-combatant` was initially added as part of the [6.1.0 release](https://github.com/FortAwesome/Font-Awesome/commits/96cafbd73ec4339b8c73c36673ce1518db82cc5c/svgs/solid/child-rifle.svg?browsing_rename_history=true&new_path=svgs/solid/child-combatant.svg&original_branch=6.x) (but was initially named `child-rifle` until 6.2.1).

Depicting a child combatant with a less-serious looking icon could be argued as detracting from the seriousness of the concept. However, I do _not_ agree that it is a joke in poor taste. This icon was added alongside many others to depict humanitarian themes in collaboration with the United Nations Office for the Coordination of Humanitarian Affairs. You can read more about it in the [release notes for 6.1.0](https://fontawesome.com/v6/changelog#:~:text=and%20duotone%20styles-,6.1.0,-Released%20March%2015).

I would recommend incorporating the principle of [Assume Good Faith](https://en.wikipedia.org/wiki/Wikipedia:Assume_good_faith) in future OSS discussion.
